src/app/Fake.Testing.ReportGenerator/ReportGenerator.fs

```diff
@@ -79,15 +93,19 @@ let private ReportGeneratorDefaultParams =
 let private buildReportGeneratorArgs parameters (reports : string seq) =
     let reportTypes = parameters.ReportTypes |> List.map (fun rt -> rt.ToString())
     let sourceDirs = sprintf "-sourcedirs:%s" (String.Join(";", parameters.SourceDirs))
-    let filters = sprintf "-filters:%s" (String.Join(";", parameters.Filters))
+    let assemblyFilters = sprintf "-assemblyfilters:%s" (String.Join(";", parameters.Filters))
+    let classFilters = sprintf "-classfilters:%s" (String.Join(";", parameters.ClassFilters))
+    let fileFilters = sprintf "-filefilters:%s" (String.Join(";", parameters.FileFilters))
 
     new StringBuilder()
-    |> StringBuilder.append (sprintf "-reports:%s" (String.Join(";", reports)))
-    |> StringBuilder.append (sprintf "-targetdir:%s" parameters.TargetDir)
+    |> StringBuilder.appendWithoutQuotes (sprintf "-reports:%s" (String.Join(";", reports)))
+    |> StringBuilder.appendWithoutQuotes (sprintf "-targetdir:\"%s\"" parameters.TargetDir)
```

Comment:
I'd prefer building a sequence/array and then using `Args.toWindowsCommandLine` which will handle escaping properly (which your quotes are not, for example when `TargetDir` ends with a backslack)

Reply:
Oh cool, that is one I didn't know of. I will fix it